Src/Support/Google.Apis.Auth/OAuth2/ComputeCredential.cs

```diff
@@ -186,6 +168,25 @@ public ComputeCredential(Initializer initializer) : base(initializer)
         IGoogleCredential IGoogleCredential.WithHttpClientFactory(IHttpClientFactory httpClientFactory) =>
             new ComputeCredential(new Initializer(this) { HttpClientFactory = httpClientFactory });
 
+        /// <summary>
+        /// Returns a task whose result, when completed, is the default service account email associated to
+        /// this Compute credential.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// This value is cached, because for changing the default service account associated to a
+        /// Compute VM, the machine needs to be turned off. This means that the operation is only
+        /// asynchronous when calling for the first time.
+        /// </para>
+        /// <para>
+        /// Note that if, when fetching this value, an exception is thrown, the exception is cached and
+        /// will be rethrown every time this property is accessed. You can create a new <see cref="ComputeCredential"/>
```

Comment:
the exception is cached and will be rethrown by the task returned by any future call to this method

Reply:
Ah yes, true.